doc/1.4/language.md

```diff
@@ -1982,14 +1982,12 @@ The *conditional* is an expression with a constant boolean value.  It
 may reference parameters declared at the same level in the object
 hierarchy, or in parent levels.
 
-The *object declarations* are any number of declarations of
-objects, session variables, saved variables, methods,
-`in each` statements or other `#if` statements, but not
-parameters or `is` statements.  When the conditional is `true`
-(or if it's the else branch of a false conditional), the object
-declarations are treated as if they had appeared without any
-surrounding *#if*.  So the two following declarations are
-equivalent:
+The *object declarations* are any number of declarations of objects, session
+variables, saved variables, methods, or other `#if` statements, but not
+parameters, `is` statements, or `in each` statements . When the conditional is
+`true` (or if it's the else branch of a false conditional), the object
+declarations are treated as if they had appeared without any surrounding *#if*.
```

Comment:
...exceot for the subtle difference for session/saved in a template. this should be mentioned somewhere, would it be too detailed to say it here?

Reply:
I say we talk about it in the template-as-types section, perhaps in a new subsection. We could present the `#if (true)` workaround and explain the cases where it is necessary.

Reply:
However, that is all unrelated to this PR.

Reply:
fair enough